test/api/node-after.js

```diff
@@ -0,0 +1,150 @@
+"use strict";
```

Comment:
ok noted, will remove them, it's a little useful for now temporarily to run the tests quickly